pytorch/inference/docker/1.11/py3/cu113/Dockerfile.gpu

```diff
@@ -145,42 +144,37 @@ RUN apt-get update \
  && rm -rf /var/lib/apt/lists/* \
  && apt-get clean
 
-RUN curl -L -o ~/miniconda.sh https://repo.continuum.io/miniconda/Miniconda3-latest-Linux-x86_64.sh \
- && chmod +x ~/miniconda.sh \
- && ~/miniconda.sh -b -p ${CONDA_PREFIX} \
- && rm ~/miniconda.sh \
- && ${CONDA_PREFIX}/bin/conda update conda \
- && ${CONDA_PREFIX}/bin/conda install -c conda-forge \
+RUN curl -L -o ~/mambaforge.sh https://github.com/conda-forge/miniforge/releases/download/4.12.0-0/Mambaforge-4.12.0-0-Linux-x86_64.sh \
```

Comment:
is there a reason this is tied to this specific version? this seems like a slight change in behavior from the prior experience - since we were previously installing "latest" every time.

Reply:
This is a good point, I will update to pull latest mamba version from Github.